config/jobs/kubernetes/kops/kops-periodics-misc.yaml

```diff
@@ -108,6 +108,41 @@ periodics:
     testgrid-dashboards: sig-cluster-lifecycle-kops, google-aws, kops-misc
     testgrid-tab-name: kops-aws-arm64
 
+- interval: 3h
+  name: e2e-kops-aws-misc-arm64-conformance
+  labels:
+    preset-service-account: "true"
+    preset-aws-ssh: "true"
+    preset-aws-credential: "true"
+  decorate: true
+  decoration_config:
+    timeout: 140m
+  spec:
+    containers:
+    - command:
+      - runner.sh
+      - /workspace/scenarios/kubernetes_e2e.py
+      args:
+      - --cluster=e2e-kops-aws-arm64-conformance.test-cncf-aws.k8s.io
+      - --deployment=kops
+      - --kops-ssh-user=ubuntu
+      - --env=KUBE_SSH_USER=ubuntu
+      - --env=KOPS_DEPLOY_LATEST_URL=https://storage.googleapis.com/kubernetes-release-dev/ci/k8s-master.txt
+      - --env=KOPS_RUN_TOO_NEW_VERSION=1
+      - --extract=ci/k8s-master
+      - --ginkgo-parallel
+      - --kops-args=--node-size=a1.large --node-image=099720109477/ubuntu/images/hvm-ssd/ubuntu-focal-20.04-arm64-server-20200609
+      - --kops-priority-path=/workspace/kubernetes/platforms/linux/amd64
+      - --kops-version=https://storage.googleapis.com/kops-ci/bin/latest-ci-updown-green.txt
+      - --kops-zones=eu-west-1a,eu-west-1b,eu-west-1c
+      - --provider=aws
+      - --test_args=--ginkgo.focus=\[Conformance\]
+      - --timeout=120m
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20200623-2424179-master
+  annotations:
+    testgrid-dashboards: sig-cluster-lifecycle-kops, google-aws, kops-misc
```

Comment:
Should this be added to some other dashboard also @lubinsz ?

Reply:
Hi @hakman 
Yes. I will added it into the arm64 conformance dashboard later.
Also, we are discussing the issue of arm64 support with sig-release/sig-node/sig-arch.
Please see the following link as reference:
https://github.com/kubernetes/kubernetes/issues/86295
If it is possible, maybe it can also be put into release-blocking page:
https://k8s-testgrid.appspot.com/sig-release-master-blocking

Reply:
Nice :). You mean weekly calls about this in sig-release/sig-node/sig-arch?
I will try to do some work to get more tests passing on ARM. Would you like to CC you on those issues and PRs?

Reply:
Yes.
We are trying to discuss this issue with upstream at the weekly meeting.
https://docs.google.com/document/d/1Fu6HxXQu8wl6TwloGUEOXVzZ1rwZ72IAhglnaAMCPqA/edit#heading=h.2pdz2xmh5ksy
https://docs.google.com/document/d/1BlmHq5uPyBUDlppYqAAzslVbAO8hilgjqZUTaNXUhKM/edit#

And please cc me on your issues and PR-s, thank you very much.

Reply:
Thank a lot. I think I will try join the calls, though the Architecture SIG is a bit late for my timezone.